### PR TITLE
feat: 🔥 Added socialRand

### PR DESCRIPTION
**Pull request by oyuutan**
Added function to provide random social networks as object =
{object.name, object.link}

✅ Closes: #113

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/falso/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A
#113 
## What is the new behavior?
Added socialRand to generate random social network!
## Does this PR introduce a breaking change?

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: packages/falso/src/tests/social.spec.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { randSocial } from '../lib/social';
+
+describe('Social', () => {
+  it('Shuould return social object in formats [{name, link}]', () => {
+    const social = randSocial();
+
+    expect(social.name != '');
```
Comment (NetanelBasal) [inline packages/falso/src/tests/social.spec.ts:7]: This isn't valid test

File: packages/falso/src/tests/social.spec.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    expect(social.link != '');
+  });
+
+  it('Should return 10 social object in formats [{name, link}]', () => {
+    const social = randSocial({ length: 10 });
+
+    expect(social.length).toEqual(10);
+
+    // Test a random element to check if it's empty
+    expect(social[3].name != '' && social[3].link != '');
```
Comment (NetanelBasal) [inline packages/falso/src/tests/social.spec.ts:17]: same

File: packages/falso/src/tests/social.spec.ts (hunk 1 above)
Comment (oyuutan) [reply packages/falso/src/tests/social.spec.ts:7]: ok sorry, I've changed the test should I make another pull request?

Comment (oyuutan) [reply packages/falso/src/tests/social.spec.ts]: I've updated test
https://github.com/ngneat/falso/pull/124/commits/1e42b5446ebaf12e09c5b6a5d78f2d8b9161b20f

File: packages/falso/src/tests/social.spec.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { randSocial } from '../lib/social';
+
+describe('Social', () => {
+  it('Shuould return social object in formats [{name, link}]', () => {
+    expect(randSocial).toBeTruthy();
+  });
+
+  it('Should generate a list of 10 social items', () => {
+    expect(randSocial({ length: 10 }).length).toEqual(20);
```
Comment (NetanelBasal) [inline packages/falso/src/tests/social.spec.ts:9]: this doesn't make sense

File: packages/falso/src/tests/social.spec.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { randSocial } from '../lib/social';
+
+describe('Social', () => {
+  it('Shuould return social object in formats [{name, link}]', () => {
```
Comment (NetanelBasal) [inline packages/falso/src/tests/social.spec.ts:4]:
```suggestion
  it('should return social object in formats [{name, link}]', () => {
```

File: packages/falso/src/tests/social.spec.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    expect(randSocial).toBeTruthy();
+  });
```
Comment (NetanelBasal) [inline packages/falso/src/tests/social.spec.ts:6]: The test doesn't reflect the "should" statement

Comment (oyuutan) [reply packages/falso/src/tests/social.spec.ts:8]: I think now the test should be ok.
https://github.com/oyuutan/falso/commit/88543249affce89a7b1cf88529b19d8cdf34475c

**Pull request merged**